### PR TITLE
[BLO-769] Make wait time dynamic to speed up QA and testing work

### PR DESCRIPTION
**Pull request by diegodelrieu**
### Issue / feature description

` Make wait time dynamic to speed up QA and testing work

### CODE REVIEW

File: packages/extension/.env.example [hunk 1]
```diff
@@ -19,3 +19,4 @@ ARGENT_X_STATUS_URL=
 #ARGENT_SHIELD_NETWORK_ID=
 #FEATURE_MULTISIG=
 #FEATURE_VERIFIED_DAPPS=
+#WAIT_TIME=
```
Comment (simonheys) [inline packages/extension/.env.example:22]: Would prefer a more verbose name - wait time for what?

Comment (dhruvkelawala) [reply packages/extension/.env.example:22]: Agree

Comment (dhruvkelawala) [reply packages/extension/.env.example:22]: It's a common term even used in [ethers.js](https://docs.ethers.org/v5/api/providers/provider/#BaseProvider)

**Review by janek26 — CHANGES_REQUESTED** [review]
does this fix the issue @canlopes is having?
This would require him to always have an "QA" build with an custom env var, not applicable to RCs

I think you should return the inflight transactions from `transactionTracker.update` instead of just a flag which indicates if there are inflight transactions.
Once done, we can calculate that flag in the method plus additionally check if there are any transactions targeting network with id `localhost`. If so, then we can acceleratepolling (aka decrease waiting time between runs)

**Comment by diegodelrieu** [comment]
> does this fix the issue @canlopes is having? This would require him to always have an "QA" build with an custom env var, not applicable to RCs
> 
> I think you should return the inflight transactions from `transactionTracker.update` instead of just a flag which indicates if there are inflight transactions. Once done, we can calculate that flag in the method plus additionally check if there are any transactions targeting network with id `localhost`. If so, then we can acceleratepolling (aka decrease waiting time between runs)

Thanks ! I think this is better now ?

**Review by canlopes — APPROVED** [review]

**Review by janek26 — APPROVED** [review]

**Pull request merged**